tests/collection/collection-layout.spec.ts

```diff
@@ -1,109 +1,130 @@
-import { test } from '../fixtures';
+import { test, expect } from '../fixtures';
 
 import { LayoutViewModeLocator, SearchOption } from '../models';
 
+
+test.beforeEach(async ({ collectionPage }) => {
+  const { collectionSearchInput } = collectionPage;
+  test.info().annotations.push({
+    type: 'Test',
+    description: 'Do collection metadata search every each test',
```

Comment:
SUGGESTION: Fix grammar. Do you mean "before each test"?

Reply:
updated